dv360proxy/dvApi.js

```diff
@@ -0,0 +1,45 @@
+const baseApi = require("./baseApi");
+
+const displayVideo360Api = function () {
+    const dvApi = baseApi.configureApiClient("https://displayvideo.googleapis.com/v1", true);
```

Comment:
I see that DV360 got updated to V2 as well. 
https://developers.google.com/display-video/api/v2-migration-guide
There is no deprecation notice yet, but, would it make sense to update beforehand?

Reply:
Done, now ping_v2 uses DV360 API V2